### PR TITLE
Add Prediction API sample.

### PR DESCRIPTION
**Pull request by jmdobry**
Adding missing Node.js sample for these docs: https://cloud.google.com/prediction/docs/developer-guide#predictionfromappengine

### CODE REVIEW

File: prediction/hostedmodels.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+var google = require('googleapis');
+var hostedmodels = google.prediction('v1.6').hostedmodels;
+
+function predict(callback) {
+  
+  google.auth.getApplicationDefault(function(err, authClient) {
+    if (err) {
+      return callback(err);
+    }
+    // The createScopedRequired method returns true when running on GAE or a
```
Comment (JustinBeckwith) [inline prediction/hostedmodels.js]: Does 'running on GAE' apply for node?  Or does this mean Managed VMs?

File: prediction/package.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache Version 2.0",
+  "scripts": {
+    "tasks": "node hostedmodels.js"
+  },
+  "dependencies": {
+    "googleapis": "^2.1.7"
+  }
+}
```
Comment (JustinBeckwith) [inline prediction/package.json]: engines field for required node version?

**Comment by JustinBeckwith** [comment]
lgtm

File: prediction/hostedmodels.js (hunk 1 above)
Comment (jmdobry) [inline prediction/hostedmodels.js]: I think it means Managed VMs, the comment came from the googleapis README

File: prediction/hostedmodels.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    // local developer machine. In that case, the desired scopes must be passed
+    // in manually. When the code is  running in GCE or a Managed VM, the scopes
+    // are pulled from the GCE metadata server.
+    // See https://cloud.google.com/compute/docs/authentication for more
+    // information. 
+    if (authClient.createScopedRequired && authClient.createScopedRequired()) {
```
Comment (jonparrott) [inline prediction/hostedmodels.js]: This really shouldn't be required, if it is, we should open a bug on the API client. Python's does this automatically.

File: prediction/hostedmodels.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+      // Scopes can be specified either as an array or as a single,
+      // space-delimited string. 
+      authClient = authClient.createScoped([
+        'https://www.googleapis.com/auth/prediction'
+      ]);
+    }
+    // Fetch the list of GCE zones within a project. 
+    // NOTE: You must fill in your valid project ID before running this sample! 
+    hostedmodels.predict({
```
Comment (jonparrott) [inline prediction/hostedmodels.js]: Can you separate the auth code away from the api client call?

File: prediction/package.json [hunk 5]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "nodejs-docs-samples-prediction",
+  "description": "Node.js samples for Google Cloud Prediction API.",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache Version 2.0",
+  "scripts": {
+    "tasks": "node hostedmodels.js"
```
Comment (jonparrott) [inline prediction/package.json]: `tasks`?

File: prediction/hostedmodels.js (hunk 3 above)
Comment (jmdobry) [inline prediction/hostedmodels.js]: ¯\_(ツ)_/¯

**Comment by jmdobry** [comment]
@jonparrott looks good?

**Comment by jonparrott** [comment]
LGTM.

**Pull request merged**